src/System.Private.Reflection.Execution/src/Internal/Reflection/Execution/EcmaFormatEnumInfoImplementation.cs

```diff
@@ -0,0 +1,106 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using global::System;
```

Comment:
We haven't gotten around to cleaning out the "global::" prefixes from the "usings" in R.Ex, but let's not introduce them in new code. They're a holdover from the days when imports were nested inside namespace declarations and the "global::" prefixes were preemptive immunization against C#'s weird rules about partial imports inside namespaces...